src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -257,6 +314,14 @@ protected String urlEncode(String str) {
     protected boolean postProcessResponse() {
         return responseIsValid();
     }
+    
+    /**
+     * Performs any put processing directly after retrieving response.
+     * @return true if all is well, false otherwise.
+     */
+    protected boolean putProcessResponse() {
```

Comment:
I did not expect this method. Would calling postProcessResponse() not be ok also. The 'post' in postProcessResponse did not really refer to http's 'POST' but more to the general meaning of 'after'...